README.md

```diff
@@ -1,2 +1,98 @@
-# publishing-tutorial-sample-repo
-A sample repo to be used in conjunction with the tutorial "Getting started publishing with Pantheon 2" 
+# Publishing tutorial sample repo
+A sample repo to be used in conjunction with the tutorial "Getting started publishing with Pantheon 2"
+
+* [Contributing to this repository](#contributing-to-this-repository)
+
+## Contributing to this repository
+
+### Filing a bug
+
+If you have any suggestions to improve or extend this repository, add a new issue in the repository.
```

Comment:
```suggestion
If you have any suggestions to improve or extend this repository, open a [new issue](https://github.com/redhat-documentation/publishing-tutorial-sample-repo/issues) in the repository.
```
I'd add the link because many teams don't track issues in their repos.

Reply:
Done.